src/views/WelcomePageView.vue

```diff
@@ -1,26 +1,50 @@
 <template>
   <HeaderComponent msg="КРОК ШИР 179dev" />
   <main class="main-content">
-    <div class="scrollable-container">
-      <p>This is the body of the home screen.</p>
-      <p>Feel free to add any content you want here!</p>
-    </div>
+    <button
+      id="create-conference"
+      class="cta-button"
+      @click="createConference"
+    >
+      Create Conference
+    </button>
   </main>
   <footer class="footer">
-    <RouterLink to="/register" class="cta-button">Register</RouterLink>
-    <RouterLink to="/log_in" class="cta-button">Log In</RouterLink>
+    <RouterLink
+      to="/register"
+      class="cta-button"
+    >
+      Register
+    </RouterLink>
+    <RouterLink
+      to="/log_in"
+      class="cta-button"
+    >
+      Log In
+    </RouterLink>
   </footer>
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
+import { defineComponent } from "vue";
+import axios from "axios";
 
-import HeaderComponent from '@/components/HeaderComponent.vue';
+import router from "@/router";
+
+import HeaderComponent from "@/components/HeaderComponent.vue";
 
 export default defineComponent({
-  components: {
-    HeaderComponent,
-  },
+    components: {
+        HeaderComponent,
+    },
+    methods: {
+        createConference() {
+            axios.post("http://0.0.0.0:8179/conference").then((response) => {
```

Comment:
Here. That's what I talked about in a comment above. The URL here also should be an environment variable.